GVFS/GVFS.Virtualization/FileSystem/FileSystemVirtualizer.cs

```diff
@@ -36,7 +37,9 @@ protected FileSystemVirtualizer(GVFSContext context, GVFSGitObjects gvfsGitObjec
 
             this.Context = context;
             this.GitObjects = gvfsGitObjects;
-            this.fileAndNetworkRequests = new BlockingCollection<FileOrNetworkRequest>();
+
+            this.highPriorityQueue = new BlockingCollection<FileOrNetworkRequest>();
+            this.lowPriorityQueue = new BlockingCollection<FileOrNetworkRequest>();
 

```

Comment:
nit - About the queue names. lowPriorityQueue and highPriorityQueue sounds generic, it hints at the existence of low and high priority operations outside the context of always-hydrate. Naming it more specific, like alwaysHydrateQueue might be more exact.

Reply:
I disagree, this is actually unrelated to the always-hydrated feature per se. We have other reasons we may want to use the low pri queue, e.g. predictive prefetching. The point here is that this class only knows that some download requests are more important than other download requests.

(That being said, this is just a review of the doc. All of the code changes are throw away, so we'll come back to a real code review later on)